src/core/model_config.proto

```diff
@@ -448,27 +448,63 @@ message ModelOptimizationPolicy
   //@@
   message ExecutionAccelerators
```

Comment:
The Graph message above with XLA should really be replaced with this interface. Can you file a ticket for that.

Reply:
DLIS-765